### ISSUE TITLE
Full screen map

### ISSUE THREAD

**Issue by Macrofig** [tableparty/table#83]
My use case might not be a popular use case but I figure I'd document it just in case.

We put the map up on a TV while we're playing in the same room. We lose a lot of real estate because of the map selector at the top. I'm aware that I can open a separate tab with the campaing link but it would be nice to have that in the DM panel as well.

<img width="1439" alt="image" src="https://user-images.githubusercontent.com/2633542/83986318-5b9cef00-a90a-11ea-8497-ec98e01488dc.png">

**Comment by jdbann** [comment]
Yeah, the map selector definitely needs to be either collapsible or hideable. We're looking at ways to handle the UI, ideally with a view to making the map full-screen.

![Example of collapsing the map selector](https://user-images.githubusercontent.com/1191840/84008714-20caa380-a96a-11ea-8eba-640629f8aa6e.gif)

I'd like to have the controls fade away when they're not in use as well, which might work nicely for a TV mode.

**Comment by jdbann** [comment]
First part of this is coming with #100
